### DISCUSSION TITLE
Should we come up with a way to either prune or hide old discussions?

### DISCUSSION THREAD

**Discussion by MikeFultonDev** [zopencommunity/meta#134] [Reference]
It doesn't look like you can 'close' them... Should we create a label of 'closed' ? Deleting them seems a bit too aggressive...

**Comment by IgorTodorovskiIBM** [comment]
I like this suggestion: https://github.com/orgs/community/discussions/3097#discussioncomment-472948 or https://github.com/orgs/community/discussions/3097#discussioncomment-494090 (prefer the second one if we can filter the top view easily)

**Reply by MikeFultonDev** [reply]
Any other thoughts?

**Comment by MikeFultonDev** [comment]
I have created a category called 'reference'. We can rename it if we don't like that. I'll move this there too in a meta way for the meta repo...

**Comment by lnlyssg** [comment]
FYI you can now close discussions - https://github.blog/changelog/2023-03-14-adding-the-ability-to-close-github-discussions/

**Reply by MikeFultonDev** [reply]
Nice. Thanks for pointing that out.
